HID-1862: CD feedback

rupl opened the pull request:
## HID-1862 / HID-1863 / HID-1847

Some minor changes based on Marina's feedback, plus restoring modal functionality in IE

rupl committed:
HDI-1862: replace links in OCHA Services

rupl committed:
HID-1862: consistent styling of submit btn on all forms

Fixed position and outline for all forms instead of just the
main search. They got bumped out of place during development
of CD Header.

rupl committed:
HID-1863: more robust logic for SVG sprite references

Having a page anchor no longer disrupts display of icons.

guillaumev commented on src/app/common/icon.js:
```diff
@@ -38,7 +38,7 @@
 
     function iconUrl (name) {
-      return $window.location.href + '#icon-' + name;
+      return $window.location.origin + $window.location.pathname + '#icon-' + name;
```
@rupl That's the thing you were talking about in the standup about SVG icons correct ?

rupl replied:
yup, using `window.location.href` and using keyboard nav to "Skip to content" meant that SVGs were pointing at something like this: http://app.hid.vm/landing#main#icon-arrow with two anchors

rupl committed:
HID-1847: avoid Number.isInteger for IE11 happiness

Using Math.floor ensures we supply an integer of milliseconds
to this function without relying on ES6 Number methods.
